micro-rdk/src/common/entry.rs

```diff
@@ -205,20 +204,36 @@ pub async fn serve_inner<S>(
         restart,
     )));
 
-    #[cfg(feature = "native")]
     let server_builder = {
+        let http2_port = 12346;
```

Comment:
This is just to get rid of naming the port twice as we did before, by using a slightly different way of constructing a`SocketAddr `.